fedimint/src/bin/configgen.rs

```diff
@@ -36,6 +36,10 @@ struct Options {
         required = true
     )]
     denominations: Vec<Amount>,
+
+    /// Federation name
+    #[clap(long = "federation-name", default_value = "Hal's trusty mint")]
```

Comment:
It seems to me we should just require it, or generate a short name from some wordlist combination, just to get people unique one.

Reply:
This is only the test config gen script, the real distributed one like in #584 could just ask the user (interesting question, who gets to decide? Is it a vote in the first BFT round? :laughing: )